requirements/deploy.txt

```diff
@@ -5,4 +5,4 @@ gunicorn==19.9.0
 django-redis-cache==2.0.0
 
 #For resizing images
-pillow==5.4.1
+pillow==6.0.0
```

Comment:
We only use this as internal requirement for doc builders not in our code. This is safe to update, here is the changelog anyway https://pillow.readthedocs.io/en/stable/releasenotes/6.0.0.html#backwards-incompatible-changes

Reply:
https://github.com/rtfd/readthedocs.org/blob/71d06f01b16168629df669173d475667f63fcf69/readthedocs/doc_builder/python_environments.py#L287-L287